src/components/Modeler.vue

```diff
@@ -327,6 +327,15 @@ export default {
     highlightNode(node) {
       store.commit('highlightNode', node);
     },
+    /**
+     * Register a mixin into a node component.
+     * Used during "modeler-before-init"
+     *
+     */
+    registerComponentMixin(component, mixin) {
+      const mixins = component.mixins === undefined ? component.mixins = [] : component.mixins;
```

Comment:
The intention of this code could be improved. You're doing a strict equality comparison of `component.mixins` to `undefined`, but what if the value happens to be `null`, or `false`, or something else?

Since we're dealing with Vue components, we can safely assume that if `mixins` is defined, it will be an array. However, there is no need for the transient `mixins` variable, as I think it hides that fact that you're mutating a property on an existing object.

I think this reads a little better:
```js
if (!component.mixins) {
  component.mixins = [];
}

component.mixins.push(mixin);
```

The "happy path" (pushing a new mixin onto the `mixins` array) remains clearly on it's own, not indented, and the "special case" of initializing an empty `mixins` array is contained in an `if` block.